src/services/dmr/DmrService.ts

```diff
@@ -159,6 +159,7 @@ export class DmrService {
       .then(
         (success: any) => deferred.resolve(success.data.result),
         (failure: any) => {
+          console.log(failure);
```

Comment:
We used to log the failure output here, however we should be handling failures in a modal (or some other means). So I would rather not reintroduce this behaviour.